apps/api/test/fixture/validSubmission.json

```diff
@@ -42,7 +42,8 @@
       ],
       "placementOptions": ["other", "anywhere"],
       "hasImmunizationTraining": true,
-      "deploymentDuration": "twoToFour"
+      "deploymentDuration": "twoToFour",
+      "hasPreviousDeployment": "yes"
```

Comment:
I think we should add this to the invalidSubmission fixture as well to keep it as the single error for firstName?